### PR TITLE
document scalafixAll InputKey

### PR DESCRIPTION
**Pull request by bjaglin**
Merge only when https://github.com/scalacenter/sbt-scalafix/pull/126 is released

### CODE REVIEW

File: docs/users/installation.md [hunk 1]
```diff
@@ -181,34 +183,17 @@ configurations. To enable Scalafix for other configuration like
...
-Optionally, add a command alias to your build to run Scalafix on your entire
-project with the shorthand `fix`
-
-```scala
-// top of build.sbt
-addCommandAlias("fix", "all compile:scalafix test:scalafix")
-```
+Both the `scalafix` & the `scalafixAll` task aggregate like the `compile`
+and `test` tasks. To run Scalafix on all projects for both main and test
+sources you can execute `all scalafix test:scalafix` or simply `scalafixAll`.
```
Comment (olafurpg) [inline docs/users/installation.md:188]: This seems like the easiest to explain option and it handles custom integrations like `it:scalafix`
```suggestion
sources you can execute `scalafixAll`.
```

Is there a difference between `all scalafixAll` and only `scalafixAll`?

File: docs/users/installation.md [hunk 2]
```diff
@@ -161,13 +162,14 @@ Great! You are all set to use Scalafix with sbt :)
...
 `myproject/test:scalafix` instead. To run on both main and test sources, execute
-`; myproject/scalafix ; myproject/test:scalafix`
+`myproject/scalafixAll`.
 
 ### Integration tests
 
 By default, the `scalafix` command is enabled for the `Compile` and `Test`
-configurations. To enable Scalafix for other configuration like
-`IntegrationTest`, add the following to your project settings
+configurations, ans `scalafixAll` will run on both of them. To enable
```
[the remark this answers is not in the document]
Comment (bjaglin) [reply docs/users/installation.md:170]: oops

File: docs/users/installation.md (hunk 1 above)
Comment (bjaglin) [reply docs/users/installation.md:188]: indeed :+1: 

> Is there a difference between all scalafixAll and only scalafixAll?

None as in both cases, configurations invocations will run independently/concurrently. I guess the latter allows a single `last scalafixAll` to see an overview of the logs easily. And the former is a command while the latter is a task obviously.  Why do you ask?

**Comment by bjaglin** [comment]
Commit amended with the suggestions :+1:

**Pull request merged**